cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-aks-sample/src/main/java/spring/cloud/azure/starter/data/cosmos/sample/aks/UserController.java

```diff
@@ -0,0 +1,80 @@
+package spring.cloud.azure.starter.data.cosmos.sample.aks;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import com.azure.cosmos.models.PartitionKey;
+
+@Controller
+@RequestMapping(path = "/users")
+public class UserController {
+
+    @Autowired
```

Comment:
Use constructor injection instead of field injection.

Reply:
done